client/components/app.jsx

```diff
@@ -28,7 +31,8 @@ class App extends React.Component {
           currentRestaurant: query.name,
         });
         return (this.getRestaurantList());
-      });
+      })
+      .catch(error => console.log(error));
```

Comment:
How to properly catch errors? Remember Toby said, just console logging isn't properly catching errors.